laze-project.yml

```diff
@@ -745,6 +745,20 @@ modules:
           - -Clink-arg=-Tkeep-stack-sizes.x
           - -Z emit-stack-sizes
 
+  - name: ram-tiny
+    help: This module marks boards with very little RAM (currently, <10KiB)
+    conflicts:
+      # disabling alloc until we have a usable use case
+      - alloc
+      # disabling until we do have a driver/stack combo that works
+      - network
+      # disabling as the default stack sizes are too small for log (defmt works)
+      - log
+    env:
+      global:
+        isr_stacksize_required_default: "512"
```

Comment:
This is a slightly dangerous thing to do. Setting these too low will still compile but then fail at runtime. The non-tiny defaults are 2Kib (ISR) and 8KiB (Executor) ...